fe/fe-core/src/main/java/com/starrocks/sql/ast/pipe/AlterPipeSetProperty.java

```diff
@@ -0,0 +1,34 @@
+// Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
You are using the wrong license header, need to correct it as starrocks' license header.

Reply:
fixed